Automattic/wp-calypso
# Free hosting trial: Add CTA to the Business plan card

Related to p1696387524534499-slack-C041RHH38NQ.

## Proposed Changes

Displays the "Try for free" CTA alongside the "Get Business" button inside the plan card:

| Pricing grid | Plan comparison grid |
| ------------ | --------------------- |
| ![image](https://github.com/Automattic/wp-calypso/assets/26530524/d3871cf5-88d9-45ee-85a8-d532fdb794d7) | ![image](https://github.com/Automattic/wp-calypso/assets/26530524/775e7fc3-89ae-41fc-b812-e4dfb40a1a19) |

I tried to devise an implementation as agnostic as possible so that we could add more trial CTAs (e.g., Commerce) to the page later (see p1696516487969099/1696387524.534499-slack-C041RHH38NQ.)

## Testing Instructions

1. Enable the `plans/hosting-trial` feature flag
2. Enter testing mode in the store admin
3. Navigate to `/setup/new-hosted-site`
4. Fill the site options step input and click continue
5. Verify that the "Try for free" CTA shows up alongside the "Get Business" CTA, both in the pricing grid and the plan comparison grid
6. Verify that as you scroll the page the trial CTA gets stuck to the page as well
7. Click the "try for free" CTA and verify that a new trial site gets created
8. Retry the flow but click the Business CTA, verify you're sent to checkout normally
